src/views/Schedule2.vue

```diff
@@ -113,6 +113,7 @@ export default Vue.extend({
   width: 185px;
 
   &.keynote {
+    min-height:65px;
```

Comment:
```suggestion
    min-height: 65px;
```
La taille est calculé en fonction de la durée, en changeant ça nous nous exposons à ce que cette carte dépasse le temps alloué et "mange" sur une autre carte en dessous 🤔

Reply:
Je m'en doutais un peu. Mais sur les sessions de 5 mn c'est vraiment super court et on ne peut pas lire, si tu as une idée smart

Reply:
Je peux augmenter le ratio l'impact est que toutes les cartes seront plus hautes